ide/coqide/coqOps.ml

```diff
@@ -749,9 +783,8 @@ object(self)
             let line_nb = start#line + 1 in
             let bol_uni = start#offset - start#line_offset in
             let bol_byte = c2b buffer s_offs bol_uni in
-            let bol_pos = bp - bol_byte in
             last_offsets <- (bol_byte, bol_uni);
-            let coq_query = Coq.add ((((phrase,edit_id),(tip,verbose)),bp),(line_nb,bol_pos)) in
+            let coq_query = Coq.add ((((phrase,edit_id),(tip,verbose)),bp),(line_nb,bol_byte)) in
```

Comment:
This is not used by the XML protocol, so this change is a no-op. I'd still be curious to see the expected spec of the locations passed to Coq.

Reply:
Yeah, not currently used.  The thought in including line_nb and bol_pos was to allow constructing a Loc.t on the server side with correct values for at least those fields in case we want to use those for error messages in the parsed text.